ui/app/components/mount-backend-form.hbs

```diff
@@ -35,18 +35,25 @@
         @modelValidations={{this.modelValidations}}
         @onKeyUp={{this.onKeyUp}}
       />
+
       <FormFieldGroups @model={{@mountModel}} @renderGroup="Method Options">
         <:identityTokenKey>
           <SearchSelectWithModal
-            @id="identityTokenKey"
+            @id="key"
             @fallbackComponent="input-search"
+            @inputValue={{@mountModel.config.identityTokenKey}}
             @onChange={{this.handleIdentityTokenKeyChange}}
             @models={{array "oidc/key"}}
             @selectLimit="1"
             @returnString={{true}}
```

Comment:
🧹  I think `returnString` is leftover from some old functionality that I think we discussed about moving to the parent (where the selection is handled depending on if it's a string or array?). I don't think I see it used anywhere 🤔